bindata/network/kuryr/monitor.yaml

```diff
@@ -0,0 +1,114 @@
+
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: kuryr-controller
+  annotations:
+    networkoperator.openshift.io/ignore-errors: ""
+  name: monitor-kuryr-controller
+  namespace: openshift-kuryr
+spec:
+  endpoints:
+  - interval: 30s
+    port: metrics
+  jobLabel: app
+  namespaceSelector:
+    matchNames:
+    - openshift-kuryr
+  selector:
+    matchLabels:
+      app: kuryr-controller
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: kuryr-controller
+  name: kuryr-controller
+  namespace: openshift-kuryr
+spec:
+  selector:
+    app: kuryr-controller
+  clusterIP: None
+  publishNotReadyAddresses: true
+  ports:
+  - name: metrics
+    port: 9102
+    protocol: TCP
+    targetPort: 9102
+  sessionAffinity: None
+  type: ClusterIP
+---
+
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: kuryr-cni
+  annotations:
+    networkoperator.openshift.io/ignore-errors: ""
+  name: monitor-kuryr-cni
+  namespace: openshift-kuryr
+spec:
+  endpoints:
+  - interval: 30s
+    port: metrics
+  jobLabel: app
+  namespaceSelector:
+    matchNames:
+    - openshift-kuryr
+  selector:
+    matchLabels:
+      app: kuryr-cni
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: kuryr-cni
```

Comment:
How does it work here? It doesn't matter which kuryr-cni gets asked for metrics? I'd say it should be somehow separated per node?

Reply:
All kuryr-cni pods get their metrics collected.